routers/sports_router.py

```diff
@@ -1,12 +1,29 @@
 from fastapi import APIRouter
+from database_connection import sport_detail_collection
 
+router = APIRouter(prefix="/sports", tags=["sports"])
 
-router = APIRouter(
-    prefix="/sports",
-    tags=["sports"]
-)
 
-
-@router.get("/sports")
+@router.get("/")
 def get_all_sports_id():
-    return {"test": "all good"}
+
+    sport_pairs = sport_detail_collection.aggregate(
+        [
+            {
+                "$replaceRoot": {
+                    "newRoot": {
+                        "$arrayToObject": [
+                            [{"k": {"$toString": "$sport_id"}, "v": "$sport_name"}]
+                        ]
+                    }
+                }
+            }
+        ]
+    )
```

Comment:
## How It Works
We aggregate from `SportDetail` collection that we only want array of key-value pairs where `sport_id` as key and `sport_name` as value.

`$replaceRoot` allows us to customize how the J/BSON output should be, by stating the root to start from inner nodes or entirely new nodes. See links below to learn more:

* https://www.mongodb.com/docs/manual/reference/operator/aggregation/replaceRoot/#-replaceroot-with-a-newly-created-document
* https://www.mongodb.com/docs/manual/reference/operator/aggregation/replaceRoot/#-replaceroot-with-an-embedded-document-field